includes/admin/class-wp-job-manager-admin-notices.php

```diff
@@ -569,6 +576,20 @@ private static function render_notice( $notice_id, $notice ) {
 		echo '</div>';
 
 	}
+
+	/**
+	 * Generate unique notice ID based on the updates available.
+	 *
+	 * @param array $updates The updates available.
+	 * @return string The notice ID.
+	 */
+	private static function generate_notice_id_from_updates( $updates ) {
+		$notice_id = self::NOTICE_ADDON_UPDATE_AVAILABLE;
+		foreach ( $updates as $update ) {
+			$notice_id .= '-' . $update['plugin'] . '@' . $update['new_version'];
+		}
+		return md5( $notice_id );
```

Comment:
Really tiny thing: what do you think about using standard prefix and then hashing the plugin/version combos? Only benefit is being able to see them together in the dismissed actions array.

Reply:
Done in 81c35791